plugins/modules/aci_l2out.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Sudhakar Shet Kudtarkar (@kudtarkar1)
+# Copyright: (c) 2020, Shreyas Srish <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l2out
+short_description: Manage layer2(L2Out) objects.
+description:
+- Manage outer layer2 on Cisco ACI fabric.
+options:
+  tenant:
+    description:
+    - Name of existing tenant.
+    type: str
+  l2out:
+    description:
+    - The name of outer layer2.
+    type: str
+    aliases: [ 'name' ]
+  description:
+    description:
+    - The description of outer layer2.
+    type: str
+  bd:
+    description:
+    - Name of the Bridge domain which is associted with L2Out.
+    type: str
+  domain:
+    description:
+    - Name of the external L2 Domain that is being associated with L2Out.
+    type: str
+  vlan:
+    description:
+    - Vlan which being associated with L2
```

Comment:
The VLAN which is being associated with the L2Out.
This should probably be a number (type int)